docker/docker-compose.ci.yml

```diff
@@ -6,6 +6,14 @@ services:
     expose:
       - 8545
     command: ["-h=0.0.0.0", "-m=hello unlock save the web"]
+  db:
+    image: postgres
+    ports:
```

Comment:
You probably only need to use `expose` here instead of `ports`. The difference IIRC is that `expose` only exposes this to other containers in the same "cluster" while `ports` will expose this outside of docker.
I think for security we should restrict this to `expose`.